MJSwiftCore/Classes/Repository/TimedCacheDataSource.swift

```diff
@@ -0,0 +1,143 @@
+//
+// Copyright 2019 Mobile Jazz SL
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+fileprivate func print(_ object: String) {
```

Comment:
I don't see value to have this code here. 

@newboadki is working in a logger protocol. We should remove this code and when @newboadki finish the logger we can update it

Reply:
goal was to let the user know that values weren't cached as queries don't conform to `KeyQuery`. However, these logs are not valuable on a production app.

Reply:
Ok.